### PR TITLE
Update index.md

### PR DESCRIPTION
**Pull request by Arawn-Davies**
Pointed newcomers to the appropriate documentation
just padded it out a wee bit 🙂

### CODE REVIEW

File: Docs/index.md [hunk 1]
```diff
@@ -1,9 +1,12 @@
...
-<br />
-
-Some docs are outdated, and we are currently working on updating them and adding new ones.
-
-<br />
-
-*Looking for the [Cosmos website](https://gocosmos.org)?*
+﻿# Welcome to the Official Cosmos Documentation</h1>
+  
+If you're knew to Cosmos, welcome! Before you get stuck in the API documentation, we highly recommend you visit the website linked at the bottom.
```
Comment (quajak) [inline Docs/index.md:3]: Typo: knew -> new

File: Docs/index.md [hunk 2]
```diff
@@ -1,9 +1,12 @@
...
-Some docs are outdated, and we are currently working on updating them and adding new ones.
-
-<br />
-
-*Looking for the [Cosmos website](https://gocosmos.org)?*
+﻿# Welcome to the Official Cosmos Documentation</h1>
+  
+If you're knew to Cosmos, welcome! Before you get stuck in the API documentation, we highly recommend you visit the website linked at the bottom.  
+  
+The Articles section of this website includes some articles which have been written many years ago for websites such as www.codeproject.com and are kept for archival purposes. These discuss elements such as the Cosmos Virtual FIle System. We prefer newcomers who want to develop their own operating systems to read the API documentation while we update these articles to comply with the current Cosmos project.
```
Comment (quajak) [inline Docs/index.md:5]: FIle  -> File

**Review by quajak — CHANGES_REQUESTED** [review]
Looks good beside the two typos!

**Comment by Arawn-Davies** [comment]
@quajak thanks for pointing those out :) 
just corrected em

**Review by quajak — APPROVED** [review]

**Pull request merged**